README.md

```diff
@@ -21,7 +21,8 @@ Police Job for QB-Core Framework :police_officer:
 
 ## Dependencies
 - [qb-core](https://github.com/qbcore-framework/qb-core)
-- [qb-bossmenu](https://github.com/qbcore-framework/qb-bossmenu) - For the boss menu
+- (Deprecated) [qb-bossmenu](https://github.com/qbcore-framework/qb-bossmenu) - For the boss menu
+- (Replace qb-bossmenu) [qb-management](https://github.com/qbcore-framework/qb-management) - For the boss/gang menu
```

Comment:
Thanks for the PR.

I think this would look clearer with the `(Deprecated)` and `(Replaces qb-bossmenu)` comments after the links.

Could you update them to be like that please?

Reply:
```lua
- [qb-bossmenu](https://github.com/qbcore-framework/qb-bossmenu)  (Deprecated) - For the boss menu
- [qb-management](https://github.com/qbcore-framework/qb-management) (Replaces qb-bossmenu) - For the boss/gang menu
```